ci/docker-ispc/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM buildpack-deps:stretch
+
+ARG ISPC_HOME=/usr/local/src/ispc
+ARG LLVM_HOME=/usr/local/src/llvm
+ARG LLVM_VERSION=8.0
+
+ENV PATH=$LLVM_HOME/bin-$LLVM_VERSION/bin:$ISPC_HOME/bin/bin:$PATH
+
+RUN set -x \
```

Comment:
heh, throwing this into a .sh file and using `COPY` to get it into the container will make this more pleasant